Feat/skip step on click

Angel-Padilla opened the pull request:
## What was done?

Remove the `skip this step` option in the quick settings menu and integrate the functionality at barometer screen by
clicking. The only stage that is not skipable is the `retractiing` stage

Angel-Padilla committed:
feat(quicksettings): show image version

The image version reported by the backend is shown at the bottom
of the quick settings item list

Angel-Padilla committed:
chore(profilhomescreen): keep all profile imgs mounted

To avoid the profile pictures to disappear in between a high speed
scroll with the dial we keep all profile images mounted

Angel-Padilla committed:
feat(barometer): add click to skip stage

While on the barometer screen, a click causes the stage to
skip, this is disabled for the 'retracting' stage

Angel-Padilla committed:
chore(quicksettings): remove 'skip this step' option

Angel-Padilla committed:
fix(barometer): transparent bubble display

When the bubble display was visible the clicks still went through it
and triggered a skip stage event

Mimoja commented:
Strongly disagree with the idea, not the code. Lets please not merge this without further discussion

Angel-Padilla committed:
chore(quicksettings): remove os status from settings option

Use an ::before pseudo class in the Styled.OptionsContainer element to render
the OsUpdate status.

This way, it does not affect the indexing and traversal of the options

optionPositionOutter and optionPositionInner now need to account for that extra
element outside of settings

Angel-Padilla committed:
chore(quicksettings): remove sw version from settings option

Use an ::after pseudo class in the Styled.OptionsContainer element to render
the OsUpdate status.

This way, it does not affect the indexing and traversal of the options

Angel-Padilla committed:
chore(quicksettings): remove unnecessary switch statements

Angel-Padilla committed:
fix(quicksettings): sw version positioning

Mimoja commented:
Just for tracking: 
The logic is fine but we are blocked by the lack of a nice UI element

semantic-release-bot committed:
chore(release): v1.93.0 [skip ci]

Mimoja committed:
feat(barometer): add placeholder animation to emphasize the skipping

Signed-off-by: Mimoja <git@mimoja.de>
